Bio/Align/__init__.py

```diff
@@ -1509,8 +1507,32 @@ class PairwiseAligner(_aligners.PairwiseAligner):
     -EVL-
     <BLANKLINE>
 
+    You can also set the value of attributes directly during construction
+    of the PairwiseAligner object by providing them as keyword arguemnts:
+
+    >>> aligner = Align.PairwiseAligner(mode='global', match_score=2, mismatch_score=-1)
+    >>> for alignment in aligner.align("TACCG", "ACG"):
+    ...     print("Score = %.1f:" % alignment.score)
+    ...     print(alignment)
+    ...
+    Score = 6.0:
+    TACCG
+    -||-|
+    -AC-G
+    <BLANKLINE>
+    Score = 6.0:
+    TACCG
+    -|-||
+    -A-CG
+    <BLANKLINE>
+
     """
 
+    def __init__(self, **kwargs):
+        super().__init__()
+        for name, value in kwargs.items():
+            setattr(self, name, value)
+
```

Comment:
This is the magic bit - short but elegant. Do you think it is self explanatory enough?

Reply:
Hmmm... One variable name change that can help, is to also use name in `def __setattr__` (next method). This will make it slightly more clear that `setattr(self ...` calls the method `def __setattr__`. 

I think, if you are familiar with the abbreviations in Python, it is self-explanatory: For every name-value pair in the keyword arguments dictionary, set the attribute 'name' of this class to 'value'. So, it depends a bit on how familiar maintainers of this project are with Python. If you never subclassed before in Python, or have never seen `setattr`, you might be confused.

Reply:
Let's add a docstring still.